infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -55,6 +55,7 @@ public class CorfuServer {
                     + "[--plugin=<plugin-config-file-path>]"
                     + "[--base-server-threads=<base_server_threads>] "
                     + "[--log-size-quota-percentage=<max_log_size_percentage>]"
+                    + "[--orthogonal-data-kb=<orthogonal_data_kb>]"
```

Comment:
The nomenclature can be improved here. I think "reserved" is better than "orthogonal".